pkg/apis/forklift/v1beta1/plan.go

```diff
@@ -93,9 +93,11 @@ type Plan struct {
 	Referenced `json:"-"`
 }
 
-// Decide if the plan should use EL9 image for virt-v2v conversion.
-// in combination with specified destination provider.
-func (p *Plan) VSphereUsesEl9VirtV2v() (bool, error) {
+// If the plan calls for the vm to be cold migrated to the local cluster, we can
+// just use virt-v2v directly to convert the vm while copying data over. In other
+// cases, we use CDI to transfer disks to the destination cluster and then use
+// virt-v2v-in-place to convert these disks after cutover.
```

Comment:
FYI @yaacov once adapting the customize for warm migration we will need to adapt it to this flow. So we would modify the image after the cutover and the `virt-v2v-in-place`.

Reply:
@jonner Could you please add this to the doc https://github.com/kubev2v/forklift/blob/main/docs/use-of-virt-v2v-in-forklift.md?

Reply:
@jonner I have added a `virt-customize` to the warm migration flow, we should add it on top of these changes. I can take over those changes but we will start using the `virt-v2v -o local`. I don't see any possible output. Can I somehow query the guest using the libguestfs tools to get the domain file?
PTAL https://github.com/kubev2v/forklift/pull/1012

Reply:
> @jonner I have added a virt-customize to the warm migration flow, we should add it on top of these changes. I can take over those changes but we will start using the virt-v2v -o local. I don't see any possible output. Can I somehow query the guest using the libguestfs tools to get the domain file?
PTAL https://github.com/kubev2v/forklift/pull/1012

please ignore that comment, did not noticed that you are using the mounted domain instead of the /var/tmp/v2v

Reply:
> hmm, I guess I can try to add something, but it seems that this document is significantly out of date already as it describes changes that are being made in forklift 2.4, and we're now on 2.6.x. So it seems that this document needs significant overhaul to be really useful, and it's not as simple as just adding some new information about the virt-v2v-in-place command...

yeah that was just NP don't need to do that, I'll need to update that doc completely